src/sv-pipeline/04_variant_resolution/scripts/process_posthoc_cpx_depth_regenotyping.py

```diff
@@ -0,0 +1,1162 @@
+                "NEW_SVLEN\tNEW_SOURCE\tNEW_START\tNEW_END\n")
+        for r in assessment:
+            new_cpx_intervals = r.new_cpx_intervals if r.new_cpx_intervals else "."
+            new_source = r.new_source if r.new_source else "."
+            new_svlen = r.new_svlen if r.new_svlen is not None else "."
+            new_start = r.new_start if r.new_start is not None else "."
+            new_end = r.new_end if r.new_end is not None else "."
+            f.write(f"{r.vid}\t{r.modification}\t{r.reason}\t{r.new_sv_type}\t{r.new_cpx_type}\t{new_cpx_intervals}\t"
+                    f"{new_svlen}\t{new_source}\t{new_start}\t{new_end}\n")
+
+
+def _parse_arguments(argv: List[Text]) -> argparse.Namespace:
+    # noinspection PyTypeChecker
+    parser = argparse.ArgumentParser(
+        description="Reassign complex variant labels based on depth regenotyping",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument('--vcf', type=str, help='Input vcf')
+    parser.add_argument('--intervals', type=str, help='BED file of genotyped intervals (gzipped)')
+    parser.add_argument('--genotypes', type=str, help='Melted depth genotypes (gzipped)')
+    parser.add_argument('--ped', type=str, help='PED family file')
+    parser.add_argument('--out', type=str, help='Output file')
+    parser.add_argument('--reclassification-table', type=str, help='Output reclassification table path', required=False)
+    parser.add_argument('--chrx', type=str, help='Chromosome X contig name', default='chrX')
+    parser.add_argument('--chry', type=str, help='Chromosome Y contig name', default='chrY')
+    if len(argv) <= 1:
+        parser.parse_args(["--help"])
+        sys.exit(0)
+    parsed_arguments = parser.parse_args(argv[1:])
+    return parsed_arguments
+
+
+def main(argv: Optional[List[Text]] = None):
+    if argv is None:
+        argv = sys.argv
+    args = _parse_arguments(argv)
+
+    all_samples, male_samples, female_samples = parse_ped(args.ped)
+    genotype_dict = parse_genotypes(args.genotypes)
+    intervals_list_dict, genotype_counts = genotype_counts_per_variant(intervals_path=args.intervals,
+                                                                       genotype_dict=genotype_dict,
+                                                                       chr_x=args.chrx,
+                                                                       chr_y=args.chry,
+                                                                       all_samples=all_samples,
+                                                                       male_samples=male_samples,
+                                                                       female_samples=female_samples)
+    genotype_counts_tree_dict = make_genotype_tree(genotype_counts)
+    cleaned_genotype_counts = list(clean_up_intervals(genotype_counts, intervals_list_dict, genotype_counts_tree_dict))
+    del genotype_counts_tree_dict
```

Comment:
Yes added those to the `del` statement. It would probably be better practice to create new functions and the scoping handle it but I do not want to make big code changes